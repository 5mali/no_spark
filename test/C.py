from datetime import datetime
import os
from os.path import dirname, abspath, join
from os import getcwd
import sys

import random
import string
import pandas as pd
import numpy as np
import torch
import socket


seed_arg = int(sys.argv[1])

seedlist = np.array([161, 314, 228, 271828, 230, 4271031, 5526538, 6610165, 9849252, 34534, 73422, 8765])
seed = seedlist[seed_arg]
random.seed(seed)
torch.manual_seed(seed)
torch.cuda.manual_seed(seed)
np.random.seed(seed)
os.environ['PYTHONHASHSEED'] = str(seed)

NAME       = 'MYPROG'
MODELNAME  = NAME + '_' + str(seed) + '.pt'
print("\nMODEL : ", NAME)
print("SEED  : ",seed_arg)
print("HOST  : ",socket.gethostname())
tic = datetime.now()

NO_OF_ITERATIONS = 1000000
for iteration in range(NO_OF_ITERATIONS):
	x = np.random.random(1)
	x = x**2+x**3

print("X = :", x)

toc = datetime.now()

print("START TIME   : ", tic)
print("END TIME     : ", toc)
print("ELAPSED TIME : ", toc-tic)
