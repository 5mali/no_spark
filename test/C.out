
MODEL :  MYPROG
SEED  :  0
HOST  :  login000
X = : [0.01050035]
START TIME   :  2019-03-01 20:20:43.047656
END TIME     :  2019-03-01 20:20:46.872643
ELAPSED TIME :  0:00:03.824987

MODEL :  MYPROG
SEED  :  1
HOST  :  login000
X = : [1.70132381]
START TIME   :  2019-03-01 20:20:43.100682
END TIME     :  2019-03-01 20:20:46.929760
ELAPSED TIME :  0:00:03.829078

MODEL :  MYPROG
SEED  :  2
HOST  :  login000
X = : [0.58876373]
START TIME   :  2019-03-01 20:20:43.085254
END TIME     :  2019-03-01 20:20:46.930529
ELAPSED TIME :  0:00:03.845275

MODEL :  MYPROG
SEED  :  3
HOST  :  login000
X = : [0.0586003]
START TIME   :  2019-03-01 20:20:43.067829
END TIME     :  2019-03-01 20:20:47.036252
ELAPSED TIME :  0:00:03.968423
